docs/resources/dataset.md

```diff
@@ -104,8 +104,8 @@ resource "satori_dataset" "dataset1" {
 Optional:
 
 - **enable_access_control** (Boolean) Enforce access control to this dataset. Defaults to `false`.
-- **enable_self_service** (Boolean) Allow users to grant themselves access to this dataset. Defaults to `false`.
-- **enable_user_requests** (Boolean) Allow users to request access to this dataset. Defaults to `false`.
+- **enable_self_service** (Boolean, Deprecated) Allow users to grant themselves access to this dataset. Defaults to `false`. The 'enable_self_service' field has been deprecated. Please use the field 'enabled' in 'satori_self_service_access_rule' resource instead.
```

Comment:
Please use the field 'enabled' in 'satori_self_service_access_rule' resource instead --> 

Please check the Dataset Permissions section in the documentation.